packages/theming/atlas/package.json

```diff
@@ -7,7 +7,10 @@
     "bump:minor": "node ./scripts/bump-minor.js",
     "bump:major": "node ./scripts/bump-major.js",
     "build": "node ./scripts/build.js",
-    "start": "node ./scripts/build.js development"
+    "start": "node ./scripts/build.js development",
+    "pretest:e2e:dev": "cross-env ATLAS_MX_PROJECT_PATH='./tests/testProject' node ./scripts/build.js",
+    "test:e2e:dev": "pluggable-widgets-tools test:e2e:web:dev",
```

Comment:
Should `--no-widget-update` be passed here?

Reply:
no this will be passed when we start using `test:e2e:web` which requires the docker setup